lib/perl/Genome/Model/Tools/Varscan/SomaticFilterWorkflow.pm

```diff
@@ -170,22 +182,16 @@ sub run_process_workflow {
     my $self = shift;
     my $input = shift;
 
-    my $w = Workflow::Operation->create(
+    my $w = Genome::WorkflowBuilder::Command->create(
         name => "Process Varscan SNVs",
-        operation_type => Workflow::OperationType::Command->get(
-            'Genome::Model::Tools::Varscan::ProcessSomatic')
+        command => 'Genome::Model::Tools::Varscan::ProcessSomatic',
     );
     $w->parallel_by('status_file');
     $self->set_lsf_queue($w);
-    $w->log_dir($self->outdir);
-    $self->validate_workflow($w);
 
-    # Launch workflow
-    $self->status_message("Launching process workflow now.");
-    my $result = Workflow::Simple::run_workflow_lsf(
-        $w,
+    my $result = $self->execute_workflow_for_operation($w, 'process', {
```

Comment:
If I did make `execute_workflow_for_operation` pass parameters by name, I'd want to do it for this whole class.  Having one subroutine that passes by name instead of position would be odd.

I think a lot of the weirdness in my change was that I kept the inline parameter passing from `run_workflow_lsf`.  Now I've moved the input hash construction to before the call.  The variable name thus is a much larger clue what is happening.

If that didn't sufficiently increase the clarity, I'd prefer to make a follow-up PR that cleans up this class more generally.  (I would probably abandon `$w` as a variable name while I'm at it!)